eth/stagedsync/stage_interhashes.go

```diff
@@ -59,7 +61,7 @@ func StageTrieCfg(db kv.RwDB, checkRoot, saveNewHashesToDB, badBlockHalt bool, t
 	}
 }
 
-func SpawnIntermediateHashesStage(s *StageState, u Unwinder, tx kv.RwTx, cfg TrieCfg, ctx context.Context, quiet bool) (libcommon.Hash, error) {
+func SpawnIntermediateHashesStage(s *StageState, u Unwinder, tx kv.RwTx, cfg TrieCfg, ctx context.Context, quiet bool, chanConfig chain.Config) (libcommon.Hash, error) {
```

Comment:
I saw this above as well `chanConfig` reads as `channel config` to me, I think maybe the intent was for this to be `chainConfig`?

Reply:
Good catch!